ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-@use "design-system";
-
-.send-screen-wrapper {
-  display: flex;
-  flex-flow: column nowrap;
-  z-index: 25;
-
-  @include design-system.screen-sm-max {
-    width: 100%;
-    overflow-y: auto;
-  }
-
-  section {
-    flex: 0 0 auto;
-  }
-}
-
-.send-screen-card {
-  background-color: var(--color-background-default);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  padding: 46px 40.5px 26px;
-  position: relative;
-  align-items: center;
-  display: flex;
-  flex-flow: column nowrap;
-  width: 498px;
-  flex: 1 0 auto;
-
-  @include design-system.screen-sm-max {
-    top: 0;
-    width: 100%;
-    box-shadow: none;
-    padding: 12px;
-  }
-}
-
-/* Send Screen */
-
-.send-screen section {
-  margin: 4px 16px;
-}
-
-.send-screen input {
-  @include design-system.H7;
-
-  width: 100%;
-}
-
-.send-eth-icon {
-  border-radius: 50%;
-  width: 70px;
-  height: 70px;
-  border: 1px solid var(--color-border-muted);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  position: absolute;
-  top: -35px;
-  z-index: 25;
-  padding: 4px;
-  background-color: var(--color-background-default);
-
-  @include design-system.screen-sm-max {
-    position: relative;
-    top: 0;
-  }
-}
-
-.send-screen-input-wrapper {
-  width: 95%;
-  position: relative;
-
-  .fa-bolt {
-    padding-right: 4px;
-  }
-
-  .send-screen-gas-input {
-    border: 1px solid transparent;
-  }
-
-  &__error-message {
-    display: none;
-  }
-
-  &--error {
-    input,
-    .send-screen-gas-input {
-      border-color: var(--color-error-default) !important;
-    }
-
-    .send-screen-input-wrapper__error-message {
-      @include design-system.H7;
-
-      display: block;
-      position: absolute;
-      bottom: 4px;
-      left: 8px;
-      color: var(--color-error-default);
-    }
-  }
-
-  .send-screen-input-wrapper__error-message {
-    @include design-system.H7;
-
-    display: block;
-    position: absolute;
-    bottom: 4px;
-    left: 8px;
-    color: var(--color-error-default);
-  }
-}
-
-.send-screen-input {
-  width: 100%;
-}
-
-.send-screen-gas-input {
-  @include design-system.Paragraph;
-
-  width: 100%;
-  height: 41px;
-  border-radius: 3px;
-  background-color: var(--color-background-default);
-  border-width: 0;
-  border-style: none;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding-left: 10px;
-  padding-right: 12px;
-  color: var(--color-text-alternative);
-}
-
-.send-screen-amount-labels {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-
-.send-screen-gas-labels {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-
-.currency-toggle {
-  &__item {
-    color: var(--color-primary-default);
-    cursor: pointer;
-
-    &--selected {
-      color: var(--color-text-default);
-      cursor: default;
-    }
-  }
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 59 04 AM](https://github.com/user-attachments/assets/24ac43d4-c017-4f73-8e39-8b5bd6ba613f)